assets/app/views/modals/update-resource.html

```diff
@@ -0,0 +1,26 @@
+<div class="modal-resource-delete">
+  <div class="modal-body">
+    <div ng-if="resourceList.length === 1">
+      <h1>{{resourceKind | humanize}} '<strong>{{resourceName}}</strong>' already exists</h1>
+      <p>Do you want to replace the current '<strong>{{resourceName}}</strong>' {{resourceKind | humanize}} with the submitted one?</p>
+    </div>
+    
+    <div ng-if="resourceList.length > 1">
+      <h1>Some of {{resourceKind | humanize}} items already exists</h1>
+      <p>Following components already exist:</p>
+      <dl class="dl-horizontal">
+        <div ng-repeat="resource in updateResources">
+          <dt>{{resource.kind}}</dt>
+          <dd>{{resource.metadata.name}}</dd>
+        </div>
```

Comment:
I think this will work:

```html
<dl class="dl-horizontal">
  <dt ng-repeat-start="resource in updateResources">...</dt>
  <dd ng-repeat-end>...</dd>
</dl>
```